zombienet/README.md

```diff
@@ -0,0 +1,23 @@
+**Zombienet for macos**
+
+Download from [zombienet](https://github.com/paritytech/zombienet/releases)
+
+or
+
+`wget https://github.com/paritytech/zombienet/releases/download/v1.3.30/zombienet-macos `
+
+`chmod +x zombienet-macos`
+
+`./zombienet-macos --help`
+
+Build Polkadot
+
+`cargo build --release`
+
```

Comment:
Maybe we should be a bit more specific here and say that you first have to git clone polkadot, then navigate to it, checkout some version and then cargo build etc